Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -1919,6 +1951,35 @@ - (void)handleInitSuccess {
     }
 }
 
+- (void) sendOpenNotificationWithOriginalURL:(NSURL*)originalURL
+                              linkParameters:(NSDictionary*)linkParameters
+                                       error:(NSError*)error {
+
+    NSMethodSignature *signature = nil;
```

Comment:
The reason is that I was showing Parth how to use invocations.

I will change it to the obvious and intuitive method.